deploy/modules/gcp/tidb-cluster/main.tf

```diff
@@ -158,4 +158,11 @@ EOS
       KUBECONFIG = var.kubeconfig_path
     }
   }
+  provisioner "local-exec" {
+    when = destroy
```

Comment:
What does terraform say when updating the tidb-cluster? Will it be recreating which means destroying and creating again?

Reply:
In this case it does not destroy and create again if tidb-cluster changes. For a `null_resource` as long as `triggers` are not specified, or the ones that are specified don't change, then it should stay the same.

Reply:
OK, cool! This is what we need.